src/server/worldserver/worldserver.conf.dist

```diff
@@ -2230,6 +2230,64 @@ Wintergrasp.NoBattleTimer = 150
 
 Wintergrasp.CrashRestartTimer = 10
 
+#
+#     TolBarad.Enable
+#         Description: Enable the Tol Barad battlefield.
+#         Default:     0 - (Disabled)
+#                      1 - (Enabled, Experimental as in incomplete, bugged and with crashes)
+
```

Comment:
kappa disable it because its incomplete and bugged and crashing the servers... :D